lib/plugins/chat.js

```diff
@@ -72,10 +72,34 @@ function inject (bot, options) {
     })
   }
 
+  function tabComplete (text, cb, assumeCommand = false, sendBlockInSight = true) {
+    let position
+
+    if (sendBlockInSight) {
+      const block = bot.blockInSight()
+
+      if (block) {
+        position = block.position
+      }
+    }
+
+    bot._client.once('tab_complete', (packet) => {
+      cb(packet.matches)
```

Comment:
callback first parameter is error, so here you need to do cb(null, packet.matches)